site/about/styles.css

```diff
@@ -45,10 +45,132 @@
     flex-direction: row;
 }
 
+.school-info-nav {
+    margin-top: 20px;
+    margin-bottom: 30px;
+}
+
 .our-mission {
     background-color: lightgray;
     padding-top: 40px;
     padding-bottom: 40px;
     padding-right: 50px;
     padding-left: 50px;
+}
+
+/* ROW 3 ONTOP */
+
+.history {
+    display: flex;
+    flex-direction: row;
+}
+
+.history-column-1 {
+    display: flex;
+    flex-direction: column;
+}
+
+.history-column-2 {
+    display: flex;
+    flex-direction: column;
+}
+
+.our-story {
+    margin-left: 50px;
+    margin-right: 50px;
+}
+
+.date-1 {
+    margin-bottom: 10px;
+}
+
+.date-3 {
+    margin-bottom: 10px;
+}
+
+.our-story-title {
+    margin: 20px 20px 20px 0;
+    font-size: 20px;
+}
+
+/* HISTORY ROW ONTOP */
+
+.box-1 {
```

Comment:
See if you could just have 1 `.box` class and use that class for all the boxes. Let me know if there are differences between any of them. Looks like the background-color may differ?